sycl/include/CL/sycl/detail/common.hpp

```diff
@@ -104,6 +104,16 @@ static inline std::string codeToString(cl_int code) {
   /* ":" __SYCL_STRINGIFY_LINE(__LINE__) ": " */                               \
                                "Native API returns: "
 
+#define __SYCL_REPORT_PLUGIN_ERR(expr, name, exc)                              \
```

Comment:
Could you please clarify if we can convert it to a function?

Reply:
will convert it.

Reply:
done